ENIGMAsystem/SHELL/Platforms/General/PFmain.cpp

```diff
@@ -23,6 +23,10 @@ int frames_count = 0;
 unsigned long current_time_mcs = 0;
 bool game_window_focused = true;
 
+// located in actions.h but not included
+// to avoid multiple definitions
+std::string game_format_caption();
```

Comment:
Stuff in actions.h is supposed to wrap stuff from the "proper" GML API. Why is this special?

Also, everything in actions.h should be small and marked `inline`.

Reply:
@JoshDreamland `GAME_GLOBALS.h` is not separating declaration from definition and included from `SHELLmain.cpp` which is why I had to do this.

Reply:
If I try to inline it and then include `actions.h` instead I get the following error.
```
In file included from [01m[KPlatforms/General/PFmain.cpp:8[m[K:
[01m[K./Universal_System/actions.h:[m[K In function '[01m[Kstd::string enigma::game_format_caption()[m[K':
[01m[K./Universal_System/actions.h:48:20:[m[K [01;31m[Kerror: [m[K'[01m[Kshow_score[m[K' is not a member of '[01m[Kenigma_user[m[K'
   48 |   if (enigma_user::[01;31m[Kshow_score[m[K && enigma_user::score != 0)
      |                    [01;31m[K^~~~~~~~~~[m[K
```

Reply:
@JoshDreamland ok 11dfaad979c9fcb1b2b4289bf8ec4d9fe8b7698c I thought why not just move them to platforms? Since that's essentially what I am doing is having the platform main loop implement them, then why not make it responsible for their definition.

Is this acceptable?